compose/ui/ui-text/src/uikitMain/kotlin/androidx/compose/ui/text/input/PlatformImeOptions.skiko.uikit.kt

```diff
@@ -0,0 +1,200 @@
+     * If not set, the value will be derived from [ImeOptions].
+     *
+     * See [UIKit documentation](https://developer.apple.com/documentation/uikit/uitextinputtraits/issecuretextentry).
+     */
+    fun isSecureTextEntry(value: Boolean?): PlatformImeOptionsConfiguration = apply {
+        isSecureTextEntry = value
+    }
+
+    /**
+     * Sets whether the return key should be automatically enabled when text is present.
+     *
+     * See [UIKit documentation](https://developer.apple.com/documentation/uikit/uitextinputtraits/enablesreturnkeyautomatically).
+     */
+    fun enablesReturnKeyAutomatically(value: Boolean): PlatformImeOptionsConfiguration = apply {
+        enablesReturnKeyAutomatically = value
+    }
+
+    /**
+     * Sets the autocapitalization style to apply.
+     * If not set, the value will be derived from [ImeOptions].
+     *
+     * See [UIKit documentation](https://developer.apple.com/documentation/uikit/uitextinputtraits/autocapitalizationtype).
+     */
+    fun autocapitalizationType(value: UITextAutocapitalizationType?): PlatformImeOptionsConfiguration = apply {
+        autocapitalizationType = value
+    }
+
+    /**
+     * Sets the autocorrection behavior to apply.
+     * If not set, the value will be derived from [ImeOptions].
+     *
+     * See [UIKit documentation](https://developer.apple.com/documentation/uikit/uitextinputtraits/autocorrectiontype).
+     */
+    fun autocorrectionType(value: UITextAutocorrectionType?): PlatformImeOptionsConfiguration = apply {
+        autocorrectionType = value
+    }
+
+    /**
+     * Builds the final PlatformImeOptions instance with the configured values.
+     */
+    internal fun build(): PlatformImeOptions {
+        return PlatformImeOptionsImpl(
+            keyboardType = keyboardType,
+            keyboardAppearance = keyboardAppearance,
+            returnKeyType = returnKeyType,
+            textContentType = textContentType,
+            isSecureTextEntry = isSecureTextEntry,
+            enablesReturnKeyAutomatically = enablesReturnKeyAutomatically,
+            autocapitalizationType = autocapitalizationType,
+            autocorrectionType = autocorrectionType,
+            hasExplicitTextContentType = hasExplicitTextContentType
+        )
+    }
+}
+/**
+ * Used to configure iOS platform IME options.
+ *
+ * Allows specifying UIKit-specific input method editor options. Note that any values
+ * explicitly specified here will override the corresponding settings from the enclosing [ImeOptions].
+ */
+fun PlatformImeOptions(configure: PlatformImeOptionsConfiguration.() -> Unit): PlatformImeOptions {
+    return PlatformImeOptionsConfiguration().apply(configure).build()
+}
+
+/**
+ * Instance that mirrors UIKit's default settings.
+ *
+ * This instance sets all properties to their UIKit default values.
+ */
+val DefaultPlatformImeOptions: PlatformImeOptions = PlatformImeOptions {
```

Comment:
Do we need this to be `public`?

Reply:
I thought it could be useful to the user if they wanted a default UIKit behavior for their inputs so that they have such API available. But they can create it themselves. I am ok with removing this altogether.